tests/internal/scripts/ccuda/cupy_arange.py

```diff
@@ -12,6 +12,6 @@ def func(a):
     threads_per_block = 32
     n_blocks = 1
     arr = cp.arange(32)
-    cuda.deviceSynchronize()
+    cuda.synchronize()
     func[n_blocks, threads_per_block](arr)
-    cuda.deviceSynchronize()
\ No newline at end of file
+    cuda.synchronize()
```

Comment:
I think this test should only call `cupy.arange` so it will only fails when needed we don't need a kernel call or the use of `cuda.threadIdx(0)` and decorators in this test

Reply:
Done in [bd6faa6](https://github.com/pyccel/pyccel/pull/1300/commits/bd6faa6f343762e484a7cea6ce6098f5a7ca5ff5).